server/redis_server.yaml

```diff
@@ -0,0 +1,9 @@
+container:
+    # Select the format for the recipe e.g. singularity, docker, podman (currently singularity is the only working option.)
+    format: singularity
+    # The image name
+    image: redis_latest.sif
+    # The url to pull the image from
+    url: docker://redis
+    # The config file
+    config: redis.conf
```

Comment:
The cool git checker is complaining about the lack of a newline here.

Reply:
Added an extra line to the end of the configuration file. I believe it should have fixed the issue.